src/vstart.sh

```diff
@@ -617,6 +635,29 @@ EOF
     done
 fi
 
+# rgw
+if [ "$start_rgw" -eq 1 ]; then
+    for rgw in `seq 0 $((CEPH_NUM_RGW-1))`
+    do
+	if [ "$new" -eq 1 ]; then
+            mkdir -p $CEPH_DEV_DIR/client.rgw.$rgw
+	    key_fn=$CEPH_DEV_DIR/client.rgw.$rgw/keyring
+	    if [ $overwrite_conf -eq 1 ]; then
+		    cat <<EOF >> $conf_fn
+[client.rgw.$rgw]
+        rgw frontends = fastcgi, civetweb port=$((CEPH_RGW_PORT+$rgw))
```

Comment:
@theanalyst - I wondered about this also, but ultimately left it as it was in the [global] section.